tests/e2e/vsan_stretched_cluster.go

```diff
@@ -55,6 +55,17 @@ var _ = ginkgo.Describe("[vsan-stretch-vanilla] vsan stretched cluster tests", f
 		storageThickPolicyName string
 	)
 
+	type siteFailure struct {
```

Comment:
we already have a struct `FaultDomains`. Under that similar to `fds.hostsDown`, we can have `downedSite` and `partitionedSite` variables. Doing it this way even if we have more than one script for vsan stretch cluster tests we will be able use the variables.